packages/gatsby-image/README.md

```diff
@@ -331,31 +331,88 @@ You will need to add it in your graphql query as is shown in the following snipp
 }
 ```
 
+## Art-directing multiple images
+
+If you want to show different images at different breakpoints (also known as [art direction](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#Art_direction), you can define your own array of `fixed` or `fluid` images, along with a `media` key per image, and pass it to `gatsby-image`'s `fixedImages` or `fluidImages` props. The `media` key that is set on an image can be any valid CSS media query.
```

Comment:
> The `media` key that is set on an image can be any valid CSS media query.

The media attribute takes a [media condition](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#The_media_attribute)(similar to a media query). Small difference that probably doesn't matter mentioning much, it's only with attributes like `sizes` where there's some limitations afaik.

Since the MDN linked docs don't really clarify the difference they're pointing out and I wasn't able to find much information elsewhere, changing `query` to `condition` doesn't need to happen, it'd possibly cause some confusion to users. 

Only mentioning it in case anyone else knows more.

Reply:
Ah, great callout. From what I can find, it seems like an entire `@media ...` expression is referred to as a query, but it's the `...` that is a media condition (or conditions).

It also looks like `media-condition` is a name used in the [formal spec](https://drafts.csswg.org/mediaqueries-4/#evaluating) for media queries.